src/framework/ui/view/navigationsection.h

```diff
@@ -36,12 +36,23 @@ namespace mu::ui {
 class NavigationSection : public AbstractNavigation, public INavigationSection, public async::Asyncable
 {
     Q_OBJECT
+    Q_PROPERTY(QmlType type READ type_property WRITE setType NOTIFY typeChanged)
+
     INJECT(ui, INavigationController, navigationController)
 
 public:
     explicit NavigationSection(QObject* parent = nullptr);
     ~NavigationSection() override;
 
+    //! NOTE Please sync with INavigationSection::Type
```

Comment:
Can we have just have one enum?

Reply:
I thought about this but didn't find a good solution.
We can add Q_ENUM for `INavigationSection::Type`, but we must register `INavigationSection` in qml, for this we must add Q_GADGET, and in Qml we must write `type: INavigationSection.Type` (maybe we can use another name into qml) I think this is not the best solution.
Alternatively, we can move all emuns to a separate namespace, which is not good for C ++ code, but is possible.
(Also we have to remember what we want to avoid with Qt in libmscore).
Perhaps we can find a tricky solution for dynamic reinstance enums from one namespace to another, but I think this is too much. The current solution is simple and works.